### PR TITLE
Feature/fgfs 101

### PR DESCRIPTION
**Pull request by MatthewWaanders**

### CODE REVIEW

File: node_modules/homey-meshdriver/lib/zwave/ZwaveDevice.js [hunk 1]
```diff
@@ -446,7 +446,11 @@ class ZwaveDevice extends MeshDevice {
 		if (capabilityGetObj.opts.pollInterval) {
 
 			let pollInterval;
+<<<<<<< HEAD
```
Comment (MatthewWaanders) [inline node_modules/homey-meshdriver/lib/zwave/ZwaveDevice.js]: Still don't know how this keeps happening, seems to be an issue with my WebStorm install

File: drivers/FGFS-101/driver.settings.compose.json [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    "children": [
+      {
+        "id": "tamper_alarm",
+        "type": "dropdown",
+        "label": {
+          "en": "Tamper alarm",
+          "nl": "Sabotage alarm"
+        },
+        "hint": {
+          "en": "This parameter (74) determines when the tamper alarm is send.",
```
Comment (RobinBol) [inline drivers/FGFS-101/driver.settings.compose.json]: sent*

File: drivers/FGFS-101/driver.settings.compose.json [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+          "nl": "Deze parameter (74) bepaald wanneer het sabotage alarm afgaat."
```
Comment (RobinBol) [inline drivers/FGFS-101/driver.settings.compose.json]: bepaalt*

File: drivers/FGFS-101/driver.settings.compose.json [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+      "en": "Nofitications",
+      "nl": "Notificaties"
+    },
+    "collapsed": true,
+    "children": [
+      {
+        "id": "low_temperature_led",
+        "type": "dropdown",
+        "label": {
+          "en": "Low Temperature LED",
```
Comment (RobinBol) [inline drivers/FGFS-101/driver.settings.compose.json]: Be consistent in capitalisation, use Title Case or don't (i'd say don't:p).

File: drivers/FGFS-101/driver.settings.compose.json [hunk 5]
```diff
@@ -0,0 +1,429 @@
...
+          "nl": "Temperatuur Laag LED"
+        },
+        "hint": {
+          "en": "This parameter (61) determines which colour the low temperature alarm LED indication displays.",
+          "nl": "Deze parameter (61) bepaald de kleur van de LED wanneer het lage temperatuursalarm afgaat."
```
Comment (RobinBol) [inline drivers/FGFS-101/driver.settings.compose.json]: bepaalt*

Comment (RobinBol) [inline drivers/FGFS-101/driver.settings.compose.json]: I think temperatuursalarm is without an s, so temperatuuralarm.

File: drivers/FGFS-101/driver.settings.compose.json [hunk 6]
```diff
@@ -0,0 +1,429 @@
...
+            }
+          }
+        ]
+      },
+      {
+        "id": "high_temperature_led",
+        "type": "dropdown",
+        "label": {
+          "en": "Type of alarm frame sent to 1st association group devices (smoke alarm)",
+          "nl": "Type alarm frame dat naar alle apparaten in eerste associatie group wordt gestuurd (rook alarm)"
```
Comment (RobinBol) [inline drivers/FGFS-101/driver.settings.compose.json]: Those titles are way too long imo:p Also the setting id does not seem to reflect what the label and hint are explaining?

**Pull request merged**